owasp-top10-2017-apps/a10/games-irados/app/routes.py

```diff
@@ -97,14 +111,38 @@ def newuser():
             message, success = database.insert_user(username, hashed_psw)
             if success == 1:
                 flash("Novo usuario adicionado!", "primary")
+                app.logger.info('remote_addr=%s path=%s method=%s message=%s' % (
+                    request.remote_addr,
+                    request.path,
+                    request.method,
+                    'User Registered',
+                ))
                 return redirect('/login')
             else:
                 flash(message, "danger")
+                app.logger.warning('remote_addr=%s path=%s method=%s message=%s' % (
+                    request.remote_addr,
+                    request.path,
+                    request.method,
+                    'Registration Failed',
+                ))
                 return redirect('/register')
 
         flash("Passwords must be the same!", "danger")
+        app.logger.warning('remote_addr=%s path=%s method=%s message=%s' % (
+            request.remote_addr,
+            request.path,
+            request.method,
+            'Registration Invalid Password',
+        ))
         return redirect('/register')
     else:
+        app.logger.warning('remote_addr=%s path=%s method=%s message=%s' % (
+            request.remote_addr,
+            request.path,
+            request.method,
+            'Registration Bad Access',
```

Comment:
As this else condition refer to `if request.method == 'POST':`, this log is including `GET /register` as well. 

When accessing this route via FE, the application is logging: 

```
[2019-02-10 23:10:24,399] WARNING in routes: remote_addr=172.25.0.1 path=/register method=GET message=Registration Bad Access
172.25.0.1 - - [10/Feb/2019 23:10:24] "GET /register HTTP/1.1" 200 -
```

Which may be a "false positive".